sdk/openai/azure-ai-openai/customization/src/main/java/ChatCompletionsToolCallCustomizations.java

```diff
@@ -32,6 +32,10 @@ public void customize(LibraryCustomization customization, Logger logger) {
                     "this.type = type;"));
         JavadocCustomization constructorJavadocCustomization = constructorCustomization.getJavadoc()
             .setParam("type", "the type value to set.");
+
+        // remove unused class (no reference to them, after partial-update)
+        customization.getRawEditor().removeFile("src/main/java/com/azure/ai/openai/models/FileDetails.java");
+        customization.getRawEditor().removeFile("src/main/java/com/azure/ai/openai/implementation/MultipartFormDataHelper.java");
```

Comment:
Use the customization to delete the class that no longer in use after partial-update.